test/css-stats.json

```diff
@@ -1,135 +1,186 @@
 {
-  "generator": "analyze-css v0.13.0",
+  "generator": "analyze-css v2.1.18",
```

Comment:
Because of the updated analyse-css version, the results here are different. I checked this branch with the old version of css-analyse and got the same results as are shown on the left (the old results), which means the CSS is the same (also checked visually of course)